tests/testthat/test_system.R

```diff
@@ -272,6 +272,11 @@ test_that("prefecturecode", {
   res <- exploratory::prefecturecode(df$kanji.with.todofuken, output_type="name")
   expect_equal(FALSE, any(is.na(res)))
 
+  df <- data.frame(a=c("北海道", "東京", "", NA, "おおさか"))
```

Comment:
Using multibyte chars in the test code literally causes errors on Windows.
Use `/uXXXX` nonation.

Reply:
Updated as suggested